x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/logs_ui_categories/ml/log_entry_categories_count.json

```diff
@@ -0,0 +1,34 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "Categorize log entries and detect count anomalies",
+  "groups": ["logs-ui"],
+  "analysis_config": {
+    "bucket_span": "15m",
+    "categorization_field_name": "message",
+    "detectors": [
+      {
+        "by_field_name": "mlcategory",
+        "detector_description": "count by mlcategory",
+        "function": "count",
+        "partition_field_name": "event.dataset",
+        "use_null": true
+      }
+    ],
+    "influencers": ["event.dataset", "mlcategory"]
+  },
+  "analysis_limits": {
+    "model_memory_limit": "100mb",
+    "categorization_examples_limit": 1
+  },
+  "data_description": {
+    "time_field": "@timestamp",
+    "time_format": "epoch_ms"
+  },
+  "model_plot_config": {
+    "enabled": true
```

Comment:
With many partitions and many categories, model plot does have a resource overhead. Note we are considering have a circuit breaker on model plot in the event of high cardinality. Would this impact any part of your UI, such as the sparklines?

Reply:
Yes, we're using the model plot to associate the categories with time even if no anomaly occurs.